### ISSUE TITLE
[Logic]: "Dungeon items: MCB + Small Keys" never passes logic check

### ISSUE THREAD

**Issue by foxsouns** [daid/LADXR#64]
Enabling MCB + Small Keys on the the latest master (`cbf5879`) does not work, due to failing logic checks. Also tested with the different Accessibility and Logic settings: no effect. Keysanity and MCB both work fine.

```
Short settings string: hsHTWRDNl,Can no longer place our item pool
Traceback (most recent call last):
  File "/home/pyodide/main.py", line 133, in main
    r = randomizer.Randomizer(args, settings, seed=userSeed)
  File "/home/pyodide/randomizer.py", line 69, in __init__
    item_placer.run(self.rnd)
  File "/home/pyodide/randomizer.py", line 226, in run
    assert self.logicStillValid(), "Sanity check failed: %s" % (self.logicStillValid(verbose=True))
AssertionError: Sanity check failed: False
```

**Comment by daid** [comment]
That was a silly mistake on my side.
